containers/bin/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apk add --no-cache go=1.13.15-r0 python3=3.8.5-r0 python2=2.7.18-r0 busybox=
 # python         3.7.4a-vZ9        /staged/positive/libpython3.7.so
 # python         3.9-aZ5           /staged/positive/patchlevel.h
 # ...
+WORKDIR /
```

Comment:
oh wow so this is now catching failures that weren't there before?

Reply:
apparently so